content/dev/code/inclusion.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Auto Inclusion System
+description: Zen Cart Auto Inclusion System
+category: code
+weight: 10
+---
+
+To reduce the number of changes required for plugin authors, Zen Cart does some automatic inclusion of files. 
+
+### Functions 
+Any functions placed in `includes/functions/extra_functions` are loaded automatically, and can be called by any storefront side logic.  Similarly, on the admin side, functions in `admin/includes/functions/extra_functions` are automatically loaded. 
+
+### Language Files 
+
+- A storefront side page will automatically load its own language file.  So the page `includes/modules/pages/contact_us/` will load `includes/languages/english/contact_us.php`
+- All files in `includes/languages/english/extra_definitions` are automatically loaded.
+
+### Extra Datafiles and Extra Configures 
+
+FIXME need content
+
+### Extra Cart Actions 
+
+FIXME need content
```

Comment:
These are identical to the `extra_functions` dir: every *.php file here is loaded automatically.